frontend/src/features/frame/components/topbar/Logo.tsx

```diff
@@ -1,37 +1,54 @@
 import React from 'react';
 
-import Grid from '@mui/material/Grid';
-import IconButton from '@mui/material/IconButton';
-import { Menu } from '@mui/icons-material';
+import { Grid, IconButton, Typography } from '@mui/material';
+
+import { useAppSelector, useAppDispatch } from 'src/app/hooks';
+import { InternalLink } from 'src/components';
+import { useCurrentPoll } from 'src/hooks';
 
 import { openDrawer, closeDrawer, selectFrame } from '../../drawerOpenSlice';
-import { useAppSelector, useAppDispatch } from '../../../../app/hooks';
-import PollSelector from './PollSelector';
-import { polls } from 'src/utils/constants';
 
 const Logo = () => {
+  const { baseUrl: pollHome } = useCurrentPoll();
   const drawerOpen = useAppSelector(selectFrame);
   const dispatch = useAppDispatch();
 
   return (
     <Grid
       item
-      md={4}
-      xs={'auto'}
-      marginRight="auto"
-      sx={{
-        display: 'flex',
-        flexDirection: 'row',
-        alignItems: 'center',
-      }}
+      xs
+      display="flex"
+      flexDirection="row"
+      alignItems="center"
+      columnGap={1}
     >
       <IconButton
         onClick={() => dispatch(drawerOpen ? closeDrawer() : openDrawer())}
         size="large"
+        sx={{
+          px: '6px',
+          py: '6px',
+          border: '1px solid rgba(0, 0, 0, 0.12)',
+          backgroundColor: 'rgba(29, 26, 20, 0.08)',
+          '&:hover': {
+            backgroundColor: 'rgba(29, 26, 20, 0.16)',
+          },
+        }}
       >
-        <Menu />
+        <img src="/svg/LogoSmall.svg" alt="Tournesol logo" />
       </IconButton>
```

Comment:
The border looks good to me. But based on the feedback we got on Discord, I think I would prefer to keep the "burger" to open/close the drawer. And move the Tournesol logo inside the `<InternalLink>`.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1976/commits/9df125e52a62321c3f4764dde85f833efb66b9f5

Reply:
Oops, I created an invalid HTML by adding a `<div>` inside a `<a>`

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1976/commits/fa322803b71493f639c2744f20fc21f3c9bc113e